modules/REST/API.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+namespace WP_Syntex\Polylang\REST;
+
+use PLL_Model;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Sets all Polylang REST controllers up.
+ *
+ * @since 3.7
+ */
+class API {
+	/**
+	 * @var PLL_Model
+	 */
+	public $model;
+
+	/**
+	 * REST languages.
+	 *
+	 * @var V2\Languages|null
+	 */
+	private $languages;
+
+	/**
+	 * Constructor.
+	 *
+	 * @since 3.7
+	 *
+	 * @param PLL_Model $model Polylang's model.
+	 */
+	public function __construct( PLL_Model $model ) {
+		$this->model = $model;
+	}
+
+	/**
+	 * Init filters and new endpoints.
+	 *
+	 * @since 3.7
+	 *
+	 * @return void
+	 */
+	public function rest_api_init(): void {
```

Comment:
```suggestion
	public function init(): void {
```
No need to repeat ourselves IMHO.

Reply:
You're right. Done in [acce644](https://github.com/polylang/polylang/pull/1505/commits/acce644159f1c493703b79be153fce4676bdc850).